### PR TITLE
Recreate CR when it is deleted

### PR DESCRIPTION
**Pull request by dmage**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1653563

### CODE REVIEW

File: pkg/operator/controller.go [hunk 1]
```diff
@@ -163,21 +162,8 @@ func (c *Controller) Handle(action string, o interface{}) {
 
 	glog.V(1).Infof("Processing %s object %s", action, objectInfo)
 
-	if cr, ok := o.(*regopapi.ImageRegistry); ok {
-		dgst, err := resource.Checksum(cr.Spec)
```
Comment (bparees) [inline pkg/operator/controller.go]: if we're not going to look at this any more, should we remove the code that's setting it?

**Comment by dmage** [comment]
Now it also fixes #93.

File: pkg/operator/controller.go (hunk 1 above)
Comment (dmage) [reply pkg/operator/controller.go]: done

**Comment by bparees** [comment]
e2e-aws-operator passed, manually merging.

**Pull request merged**